src/components/Button/IconButton/IconButton.module.css

```diff
@@ -21,50 +21,23 @@ limitations under the License.
 
   inline-size: var(--cpd-icon-button-size);
   block-size: var(--cpd-icon-button-size);
-  padding: calc(var(--cpd-icon-button-size) * 0.125);
+  padding: 0px;
```

Comment:
I don't really understand why this padding was here and it was throwing things off. Any insights welcome but as far as I can see it was unnecessary.

Reply:
Never mind, see second commit.